packages/mobile-top-nav/src/assets/img/media-menu-images.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable */
 import { LitElement, html } from 'lit-element';
+import * as icons from './icons.js';
 
 class mediaMenuImage extends LitElement {
```

Comment:
What is this being used for? I don't see it in the diff

Reply:
It's being rendered in `<media-menu>`